README.md

```diff
@@ -77,6 +60,47 @@ yarn add --dev prettier @prettier/plugin-pug
 yarn prettier --write "**/*.pug"
 ```
 
+### Selectively ignoring automatic formatting
+
+You can disable code formatting for a particular element by adding <nobr>`//- prettier-ignore`</nobr> comments in your pug templates:
+
+```pug
+div.text( color =   "primary",  disabled  ="true"  )
+// -prettier-ignore
+div.text( color =   "primary",  disabled  ="true"  )
+// -prettier-ignore
```

Comment:
```suggestion
//- prettier-ignore: just leave it as is
```

If you have a better _reason_ sentence, you are free to use it :slightly_smiling_face:

Reply:
I quite like `  //- prettier-ignore: because reasons` :wink

Reply:
https://mltshp-cdn.com/r/EXAW

Reply:
![EXAW](https://user-images.githubusercontent.com/213275/95002997-996d0400-05da-11eb-99ae-a0ebd0fcde78.png)